File: carverify_frontend/src/models/People.ts
class People {
  id: number;

  name: string;

  address: string;

  neighborhood: string;

  age: number;

  nmHouse: number;

  city: string;

  constructor(
    id = 0,
    name = '',
    address = '',
    neighborhood = '',
    age = 0,
    nmHouse = 0,
    city = '',
  ) {
    this.id = id;
    this.name = name;
    this.address = address;
    this.neighborhood = neighborhood;
    this.age = age;
    this.nmHouse = nmHouse;
    this.city = city;
  }
}

export default People;

File: carverify_backend/CarVerifyApp.Core/Model/People.cs
using System.ComponentModel.DataAnnotations;
using System.Diagnostics.CodeAnalysis;

namespace CarVerifyApp.Core.Model
{
    [ExcludeFromCodeCoverage]
    public class People
    {
        [Key]
        public int Id { get; set; }
        [Required(ErrorMessage = "Name is required.")]
        public string Name { get; set; }

        [Required(ErrorMessage = "Age is required.")]
        public int Age { get; set; }

        [Required(ErrorMessage = "Address is required.")]
        public string Address { get; set; }

        [Required(ErrorMessage = "NmHouse is required.")]
        public int NmHouse { get; set; }

        [Required(ErrorMessage = "Neighborhood is required.")]
        public string Neighborhood { get; set; }

        [Required(ErrorMessage = "City is required.")]
        public string City { get; set; }
    }
}
